src/components/atom/button/button.tsx

```diff
@@ -0,0 +1,11 @@
+import { ButtonProps } from './types';
```

Comment:
For consistency, please rename (update to uppercase) this folder and file to `/Button/Button.tsx`

Reply:
copy, I forgot to apply camel casing, will be applying it to all files now.